drivers/ieee802154/ieee802154_rf2xx_iface.c

```diff
@@ -29,27 +29,27 @@ void rf2xx_iface_phy_rst(struct device *dev)
 	const struct rf2xx_context *ctx = dev->driver_data;
 
 	/* Ensure control lines have correct levels. */
-	gpio_pin_write(ctx->reset_gpio, conf->reset.pin, 1);
-	gpio_pin_write(ctx->slptr_gpio, conf->slptr.pin, 0);
+	gpio_pin_set(ctx->reset_gpio, conf->reset.pin, 0);
+	gpio_pin_set(ctx->slptr_gpio, conf->slptr.pin, 0);
 
 	/* Wait typical time of timer TR1. */
 	k_busy_wait(330);
 
-	gpio_pin_write(ctx->reset_gpio, conf->reset.pin, 0);
+	gpio_pin_set(ctx->reset_gpio, conf->reset.pin, 1);
 	k_busy_wait(10);
-	gpio_pin_write(ctx->reset_gpio, conf->reset.pin, 1);
+	gpio_pin_set(ctx->reset_gpio, conf->reset.pin, 0);
```

Comment:
Please, make sure to keep original sequence otherwise radio will be at RESET state.

Reply:
@nandojve I believe this is correct.  Because the reset pin is marked active high and `gpio_pin_set()` operates on the active level, the command before the wait enables reset, and the one after disables it.

Reply:
Understood, but reset is active low see 1.1 Pin Descriptions on the datasheet. If this is on code need to be fixed anyway. I will have access to radios again next week and I can go in deep. How urgent is this change?

Reply:
Yes, I mis-wrote: reset is active low in its devicetree definition, so passing 1 to `gpio_pin_set()` makes it low.  This is different from `gpio_pin_set_raw()`.

Urgency: This will be merged today or tomorrow to enable a major refactoring required to get this branch into 2.2.

Reply:
Ok, please, move forward!